.eslintrc

```diff
@@ -0,0 +1,156 @@
+{
+	"env": {
+		"browser": true
+	},
```

Comment:
We'll probably want to clean this up in a similar way to wordpress/grunt-patch-wordpress#72 -- consume the core eslint plugin, etc -- but that's a follow-up, not necessary to resolve for this PR.

Reply:
Pulled this out into #3 so we can table it for now and revisit once the important stuff is done :ok_hand: